docs/ruleset_functions/get_largest_exhaust_source_cfm.md

```diff
@@ -0,0 +1,39 @@
+# get_largest_exhaust_source_cfm  
+
+**Description:** Returns the largest exhaust source cfm in the RMI sent to this function. This is for evaluating G3.1.2.10 exception 6 which states "Where the largest exhaust source is less than 75% of the design outdoor airflow. This exception shall only be used if exhaust air energy recovery is not used in the proposed design. "
```

Comment:
I think the 75% criteria should be checked based on the zones served by a given baseline system, not for the entire RMI.
Also, I think we only need to be looking at the baseline design, not the proposed design.